api/src/functionalTest/java/uk/gov/hmcts/reform/divorce/caseapi/SaveAndSignOutNotificationTest.java

```diff
@@ -120,6 +134,56 @@ public void shouldFailValidationErrorWhenEmailIsMissing() {
                 + "\"message\":\"email_address is a required property\"}]");
     }
 
+    @Test
+    public void shouldReturn401WhenServiceAuthTokenIsInvalid() {
+        Response response = RestAssured
+            .given()
+            .relaxedHTTPSValidation()
+            .baseUri(testUrl)
+            .header(CONTENT_TYPE, APPLICATION_JSON_VALUE)
+            .header(SERVICE_AUTHORIZATION, "Bearer invalid_token")
+            .body(
+                CcdCallbackRequest
+                    .builder()
+                    .caseDetails(
+                        CaseDetails
+                            .builder()
+                            .caseData(caseData())
+                            .build()
+                    )
+                    .build()
+            )
+            .when()
+            .post(SAVE_AND_SIGN_OUT_CALLBACK_URL);
+
+        assertThat(response.getStatusCode()).isEqualTo(UNAUTHORIZED.value());
+    }
+
+    @Test
+    public void shouldReturn403WhenServiceIsNotAuthorised() {
```

Comment:
as with the previous comment we can probably have a generic check for 403 response rather than doing it on every single endpoint